.slashrc

```diff
@@ -210,6 +211,7 @@ class MediaPlugin(slash.plugins.PluginInterface):
     self.test_call_timeout = 0
     #self._set_test_details(**variation)
     result.data.update(test_start = dt.now())
+    assert self._check_case_in_black_list()
```

Comment:
any hang message that occurs after last test_end and before next test_start will be captured in current test.

Reply:
test_start is called before the actual test functions are run.

Reply:
Yes, if an assert exception is raised before test_start ends, then the test is aborted and marked as failed and does not execute any of our actual test function code.  Isn't that what we want... to abort blacklisted cases early and mark them as failed?

Reply:
How can step 1 cause the system to hang if it's not running any of our middleware command lines?  That just seems strange to me.

The slash test logic should be:
1. load test case definition
2. call test_start hook
3. call test run function (e.g. our internally defined functions that begin with `test_` prefix)
4. call test_end hook

Inside step 3 is where we call middleware command-lines.

If step 1 is causing system hang then it is not related to any specific test case... it would have to be some internal bug to slash or other general library code and can't be addressed via blacklisting.

Reply:
> If step 1 is causing system hang then it is not related to any specific test case... it would have to be some internal bug to slash or other general library code and can't be addressed via blacklisting.

...the only way I can see issue with step 1 is if previous test case introduced a latent hang.  In that case, the previous case should be blacklisted.